test/libsolutil/UTF8.cpp

```diff
@@ -104,7 +106,7 @@ hélló
 
 ʰ ʱ ʲ ʳ ʴ ʵ
 
-̀ ́ ̂ ̃ ̄ ̅
```

Comment:
please undo

Reply:
that must have been an accident... or maybe my editor.. will revert